### DISCUSSION TITLE
MIT/Apache 2.0

### DISCUSSION THREAD

**Discussion by ryo33** [jsinger67/parol#30] [Ideas]
In the Rust community, a dual license under MIT/Apache 2.0 is common, and Rust itself is so.
A typical example is Bevy, which was initially MIT licensed, but [they worked hard to relicense it](https://github.com/bevyengine/bevy/issues/2373).

The reason why the dual-license is reasonable is discussed in https://github.com/bevyengine/bevy/issues/2373, and it says:

> 1. The MIT license (arguably) requires binaries to reproduce countless copies of the same license boilerplate for every MIT library in use. MIT-only engines like Godot [have complicated license compliance rules as a result](https://docs.godotengine.org/en/stable/tutorials/legal/complying_with_licenses.html)
> 2. The Apache-2.0 license has protections from patent trolls and an explicit contribution licensing clause.
> 3. The Rust ecosystem is largely Apache-2.0. Being available under that license is good for interoperation and opens the doors to upstreaming Bevy code into other projects (Rust, the async ecosystem, etc).
> 4. The Apache license is incompatible with GPLv2, but MIT is compatible.

Fortunately for parol, the number of contributors has not exploded yet, so that it will be easy.

**Comment by jsinger67** [comment]
I already considered a dual license but haven't done it yet.
Now I think it's a good time to do it. I will add Apache 2.0 as an alternative license to parol.

**Comment by jsinger67** [comment]
Fixed with 0ccc0ba and 59e42b0
